packages/config/src/layer2s/common/stages/getStage.ts

```diff
@@ -39,7 +39,7 @@ const getBlueprint = (opts?: GetStageOptions) =>
           positive:
             'A source-available node exists that can recreate the state from L1 data.' +
             (opts?.rollupNodeLink
-              ? ` [View code](${opts.rollupNodeLink})`
+              ? ` [View code](${opts.rollupNodeLink}).`
```

Comment:
I am not sure whether we need this dot at the end
![image](https://github.com/l2beat/l2beat/assets/72789647/49af1187-cb19-4273-a706-3b5c4825f26b)

Reply:
yeah looks a bit weird, but there is a test failing so maybe we should fix that